build/cmake/functions.cmake

```diff
@@ -601,7 +601,7 @@ function(wx_add_thirdparty_library var_name lib_name help_str)
         # If the sys library can not be found use builtin
         find_package(${lib_name})
         string(TOUPPER ${lib_name} lib_name_upper)
-        if(NOT ${${lib_name_upper}_FOUND})
+        if(NOT ${lib_name_upper}_FOUND AND NOT ${lib_name}_FOUND)
```

Comment:
I just wonder, what is the CMake convention here in general? Is it supposed to always accept both `libfoo_FOUND` and `LIBFOO_FOUND` interchangeably?

Reply:
It depends on the package name. For example [wxWidgets](https://cmake.org/cmake/help/latest/module/FindwxWidgets.html) has `wxWidgets_FOUND`, and [jpeg](https://cmake.org/cmake/help/latest/module/FindJPEG.html) has `JPEG_FOUND`.
The packages we used so far all have capitalized names, but NanoSVG probably looks like `NanoSVG_FOUND`.

Reply:
So what you're saying is that there is no real standard/convention?

Reply:
I can't find any convention, they use `<PackageName>_FOUND` found in the documentation.

But I think we could remove `${lib_name_upper}_FOUND` since all names are already in the correct capitalization.